script.js

```diff
@@ -0,0 +1,165 @@
+const toDoForm = document.querySelector("form");
+const toDoInput = toDoForm.querySelector(".input-text");
+const toDoButton = toDoForm.querySelector(".input-button");
+
+const TODO_DB = "todo";
```

Comment:
지금은 코드가 간단하여 이해가 쉽겠지만, 코드가 복잡해지고 어플리케이션의 규모가 커질수록 변수의 이름을 **명확**하고 **자세**하게 그리고 **일관적**으로 지어주는 것이 중요합니다!

여기서 `TODO_DB`라고 한다면, 의미가 모호해질 수 있습니다. 
`TODO_DB`라는 이름보다는 "`localStorage`의 접근 키"라는 사실을 더 잘 드러낼 수 있게 다음과 같이 고쳐보는 것은 어떨까요?

예를 들면,
```suggestion
const LOCAL_STORAGE_TODO_KEY = "todo";
```

Reply:
코드가 간단해도 이러한 습관을 들여야겠군요:)